apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -42,6 +53,20 @@ internal static class Labels
         ///<summary>The label to denote a stack trace.</summary> 
         internal const string StackTrace = "trace.cloud.google.com/stacktrace";
 
+        ///<summary>The label to denote an agent.</summary> 
+        internal const string Agent = " /agent";
+
+        /// <summary>
+        /// Gets a map with the label for the agent which contains the agent's name and version.
```

Comment:
Nothing's going to mutate this, are they? If not, it would be better to make this a static property, I suspect. There's no point in creating a new dictionary every time.